pkgs/tools/package-management/lix/default.nix

```diff
@@ -279,7 +198,6 @@ lib.makeExtensible (self: {
           self.${version}.lix;
     in
     lib.dontRecurseIntoAttrs {
-      lix_2_91 = mkAlias "lix_2_91";
```

Comment:
Dropping this made me wonder about whether we should add aliases-style information for users.
Any opinions from nixpkgs developers?
cc @emilazy

Reply:
@emilazy in aliases.nix or in this file?